packages/react/src/SelectPanel/SelectPanel.module.css

```diff
@@ -53,19 +53,10 @@
 }
 
 .Notice {
-  display: flex;
-  padding-top: var(--base-size-12);
-  padding-right: var(--base-size-16);
-  padding-bottom: var(--base-size-12);
-  padding-left: var(--base-size-16);
   margin-top: var(--base-size-4);
   margin-right: var(--base-size-8);
   margin-left: var(--base-size-8);
-  font-size: var(--text-body-size-small);
-  flex-direction: row;
-  border: var(--borderWidth-thin) solid;
-  gap: var(--base-size-8);
-  border-radius: var(--borderRadius-medium);
+  padding: var(--base-size-4) !important;
```

Comment:
open to suggestions on how to avoid this !important 😅 , the Banner class is adding way too much padding for this use case